### DISCUSSION TITLE
Nextcloud AIO Files not creating - log shows client POST error 403 forbidden response

### DISCUSSION THREAD

**Discussion by brendongl** [nextcloud/all-in-one#5771] [Questions] [answered]
I have Nextcloud AIO installed on unraid.
I'm using cloudflare zero tunnel route and most of it seems to have worked except files and transfer speeds.

I followed all the steps in https://www.youtube.com/watch?v=U47nvwXrAOo
It was working at once point - but now it stopped.

Client error: `POST https://domainname.com/cool/extract-document-structure?filter=contentcontrol` resulted in a `403 Forbidden` response 
![image](https://github.com/user-attachments/assets/a0f3184f-1ddf-4465-8a7b-488d25b0163f)

![image](https://github.com/user-attachments/assets/7434ffe7-a10c-4538-9659-89481d2c8557)
![image](https://github.com/user-attachments/assets/2cb4a729-c796-4d04-9607-2d6e399b8d44)
![image](https://github.com/user-attachments/assets/379643c3-2b1e-44e3-baf2-94d8568366dd)

**Comment by szaimen** [comment, marked as answer]
Hi, I suppose you are running into this: https://github.com/nextcloud/all-in-one/issues/5763
